src/app/QuickStarts/quickstarts/settings-quickstart.tsx

```diff
@@ -65,16 +65,16 @@ The **Connectivity** tab enables you to configure the WebSocket connection betwe
 
 1. Click [Connectivity]{{highlight settings-connectivity-tab}}.
 2. Configure the **WebSocket Connection Debounce** time.
-3. Configure the **Auto-Refresh** period for content-views.
-[To use the **Auto-Refresh** feature, make sure to enable the [Auto-Refresh]{{highlight settings-connectivity-tab-auto-refresh}} checkbox.]{{admonition tip}}
+3. Configure the **Auto-refresh** period for content-views.
```

Comment:
Should this just be "auto-refresh", all lowercase?

Reply:
This is now matching the updated UI `Auto-refresh`. the `A` is capitalized because its at the beginning.